styles/ui-variables.less

```diff
@@ -0,0 +1,85 @@
+// This file has fallback variables. It specifies the variables themes
```

Comment:
I'm guessing this is what is causing less compiler problem? If so, I'm not against adding it, but it seems counter productive to me as these styles may not work with the user selected theme.

Do you have an example of other projects including this file?

Reply:
If this is the case, we can remove it. On Atom Dark without this file, the spacers were hidden, and I couldn't build the less file. 
No I don't know what others do. Maybe there is another way.

Reply:
I added a screenshot the OP that shows the problem that is solved.

Reply:
> I added a screenshot the OP that shows the problem that is solved.

Thanks, that clarifies your motivation :smile: 

> If this is the case, we can remove it. On Atom Dark without this file, the spacers were hidden.

For your information, you can fix that using [your personal stylesheet](https://flight-manual.atom.io/using-atom/sections/basic-customization/#style-tweaks). I understand that you  might want to fix it for everyone using Atom Dark, but in that case I would create a fix in Atom Dark.


> and I couldn't build the less file.
>
> No, I don't know what others do. Maybe there is another way to link to Atom's default theme instead. Since less doesn't have a compiler config here, I don't know how it is possible. Maybe there is a CLI command for that.

It is not clear to me why you want to compile the less. As far as I know you don't need to do that as Atom will compile it for you at runtime, and it doesn't seem to run in to any problems doing that.

Reply:
So does the Atom's compilation result change depending on the active theme?